probatus/feature_elimination/feature_elimination.py

```diff
@@ -526,11 +526,12 @@ def fit(
             # Get current dataset info
             current_features_set = remaining_features
             if columns_to_keep is None:
-                remaining_removeable_features = sorted(list(set(current_features_set)))
+                remaining_removeable_features = pd.Series(current_features_set).unique()
             else:
-                remaining_removeable_features = sorted(
-                    list(set(current_features_set) | set(columns_to_keep))
-                )
+                remaining_removeable_features = pd.Series(
```

Comment:
Would it be nice to comment that this is for freezing the order?

Reply:
Fixed.